src/features/events/components/newsletter/newsletter.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import NewsletterForm from './'
+
+describe('<NewsletterForm/>', () => {
+  beforeAll(() => {
+    global.___loader = {
+      enqueue: jest.fn(),
+    }
```

Comment:
Why does this require mocking out?

Reply:
Deleted tests per our conversation about this component being deprecated for time being